src/app/static/src/js/components/runReport/changeLog.vue

```diff
@@ -63,8 +66,21 @@
                 required: true,
                 type: Boolean
             },
-            changelogStyle: Object,
-            changelogTypeOptions: []
+            changelogStyleReport: {
+                required: false,
+                type: Boolean,
+                default: false
+            },
+            changelogTypeOptions: Array
+        },
+        computed: {
+            displayStyle() {
+                if (this.changelogStyleReport) {
+                    return {label: "col-sm-2 text-right", control: "col-sm-6"}
+                }
+
+                return {label: "col-sm-4 text-left", control: "col-sm-4"}
+            }
```

Comment:
I much preferred the way this used to work i.e. the parent specifying how the widget is styled. Was there a problem with that approach?

Reply:
I really don't see anything wrong having the css class returned this way. Your preference is a matter of choice.

Reply:
It's a question of responsibility - it would be better if a child component doesn't need to know all the contexts in which it is going to be used and style itself appropriately. The previous way was more flexible for future uses which may require slightly different width/justify formatting, which could be introduced without needing to change the changelog component.

Reply:
done()